### PR TITLE
personal_* meta functions

### LINKED ISSUE #13: Add personal_* account meta methods
https://github.com/ethcore/parity/pull/1695

### PR DESCRIPTION
**Pull request by jacogr**
close #13

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1254,6 +1254,29 @@ var net = {
 };
 
 var personal = {
+  accountsInfo: {
+    desc: 'returns a map of accounts as an object',
```
Comment (tomusdrw) [inline index.js:1258]: as an array?

Comment (jacogr) [inline index.js:1258]: It comes back as a map, looks like this -

```
{
  '0xaddress1': { name: 'some name', uuid: 'some uuid', meta: 'json string' },
  '0xaddress2': ...
}
```

File: index.js [hunk 2]
```diff
@@ -1254,6 +1254,29 @@ var net = {
...
+    params: [],
+    returns: {
+      type: Array,
```
Comment (jacogr) [inline index.js:1261]: Aha, this is wrong then...

Comment (keorn) [reply index.js:1261]: Its the only method that returns a map, should we change it to array with another key being address or add another type?

Comment (jacogr) [reply index.js:1261]: I would not touch the implementation this close to the 1.4 release, the UI uses it extensively. (It does make things easier, once you get past the fact that it is "non-standard")

**Pull request closed without merging**